dependencies.yaml

```diff
@@ -38,6 +38,7 @@ files:
       - develop
       - bench
       - bench_python
+      - rapids_build_setuptools
```

Comment:
Typically we only use these lists as build-system extras in a pyproject.toml. Why is it going in a conda environment here?